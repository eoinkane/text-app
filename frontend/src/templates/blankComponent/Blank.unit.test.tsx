import React from "react";
import { shallow } from "enzyme";
import Blank from "./index";

describe("Blank", () => {
  it("Renders ", () => {
    const blank = shallow(<Blank />);
    expect(blank).toMatchSnapshot();
  });
});
