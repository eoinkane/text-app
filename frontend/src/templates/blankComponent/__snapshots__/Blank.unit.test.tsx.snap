// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blank Renders  1`] = `<Fragment />`;
